quartz/styles/base.scss

```diff
@@ -513,3 +513,15 @@ ol.overflow {
     padding-left: 1rem;
   }
 }
+
+.katex-display {
+  overflow-x: auto;
+  overflow-y: visible;
```

Comment:
visible means it will always try to show a scrollbar, would just having `overflow-x` not suffice?

Reply:
`overflow: visible` is default.

applying `auto` to `overflow-x` also applies the same to `overflow-y`. By setting `overflow-y: visible`, I restore the default behavior for the vertical overflow.

https://developer.mozilla.org/en-US/docs/Web/CSS/overflow#:~:text=The%20overflow%20property,the%20vertical%20direction.

Reply:
til haha thats so weird